### ISSUE TITLE
javascript crash in grade page

### ISSUE THREAD

**Issue by birdy4012** [wiris/moodle-atto_wiris#29]
Hi,
we have a issue in grade page.
Please, see Moodle tracker for details here:
[https://tracker.moodle.org/browse/MDL-61418](https://tracker.moodle.org/browse/MDL-61418)
Also, you will find a video.

**Comment by manuelwiris** [comment]
@birdy4012 

Thanks for your feedback. We have created an issue in our tracker, once it's fixed we will get back to you.

Best,

**Comment by birdy4012** [comment]
Hi,
if this can help you :
we are testing Moodle 3.5.0. This new version has file loader in course parameters edit page (/course/edit.php?id=123).
When we update Wiris from 7.0 to 7.3 (exactly 7.0.0.1386 to 7.3.0.1389, 2018060400), file loader doesn't work anymore.
When we remove "math = wiris" from Atto bar, every javascript next the wysiwyg field work again.
We had found that since Wiris 7.1 in every page with the Atto wysiwyg field followed by any block using javascript (like restriction access block for a course section, where the field displays json content instead of UI button).
The Chrome console displays:
![image](https://user-images.githubusercontent.com/9442042/41252097-2c0b882c-6d8a-11e8-8817-0183670bf699.png)
Best

**Comment by manuelwiris** [comment]
Hi @birdy4012 

We can't reproduce your issue: following the steps in MDL-61418, the file picker loads OK: 

![image](https://user-images.githubusercontent.com/10197199/41340269-436e0f08-6ef7-11e8-8f83-3100f913e3cf.png)

Here the configuration of the assignment configuration: 

![image](https://user-images.githubusercontent.com/10197199/41340306-5aa0659a-6ef7-11e8-8358-da3bf56e9078.png)

![image](https://user-images.githubusercontent.com/10197199/41340321-62c75e68-6ef7-11e8-8468-6ef9bba9f90f.png)

I'm guessing if you have some extra Atto plugin which, in some way, produces the error if Wiris for Atto is installed. 

If not: can you provide us more details about the issue? It would be useful if you can provide us with a backup of the activity. 

Best,

**Comment by birdy4012** [comment]
Hi Manuelwiris,
I found the When (but not the Why :-) ).
With collapse tool (_collapse = collapse_ in _editor_atto|toolbar_), when its limit (_atto_collapse|showgroups_) is >= than the number of tool, so when the collapse button is not display, AND Wiris is in the buttons list : following JS is down.
If I remove "_math = Wiris_" OR "_collapse = collapse_" from Atto toolbar, OR if I set collapse limit to a value less than the number of button group : all next JS in page work.

NB: I tested on the course parameter page (Mdl 3.5.0). Description section have a file loader block.

Brgs
